lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -2615,6 +2617,48 @@ Value *PatchInOutImportExport::patchFsBuiltInInputImport(Type *inputTy, unsigned
       input = builder.CreateInsertElement(input, kCoord, uint64_t(1));
       break;
     }
+    case PrimitiveType::Triangle_Strip:
+    case PrimitiveType::Triangle_Fan:
+    case PrimitiveType::Triangle_Strip_Adjacency: {
+      // Odd
+      auto kCoord = builder.CreateFSub(ConstantFP::get(Type::getFloatTy(*m_context), 1.0), iCoord);
+      kCoord = builder.CreateFSub(kCoord, jCoord);
+      Value *odd = UndefValue::get(inputTy);
+      odd = builder.CreateInsertElement(odd, iCoord, uint64_t(2));
```

Comment:
I think the `uint64_t` C-style cast is unneeded here and below.

Reply:
It will report an error "ambiguous call to overloaded function" without C-style cast.
PS:
it is right like this:
  input = builder.CreateInsertElement(input, iCoord, uint64_t(0));
  input = builder.CreateInsertElement(input, jCoord, 1);
  input = builder.CreateInsertElement(input, kCoord, 2);

but it is wrong like this:
  input = builder.CreateInsertElement(input, iCoord, uint64_t(2));
  input = builder.CreateInsertElement(input, jCoord, 0);
  input = builder.CreateInsertElement(input, kCoord, 1);

Reply:
Oh, I see. `0` is ambiguous between an `uint64_t` and a `Value*` because it can be interpreted as `null`. Thanks for checking.

(Seems like numbers other than 0 are not ambiguous and the ambiguity of 0 can also be solved by writing `0ul`, but that’s better done as a separate change.)